IMAGE:
  WIDTH: 640
  HEIGHT: 480
  ROI_START_HEIGHT: 365
  ROI_HEIGHT: 45
  //roi set

XYCAR:
  START_SPEED: 0.0
  MAX_SPEED: 28.0
  MIN_SPEED: 13.9
  SPEED_CONTROL_THRESHOLD: 10.0
  ACCELERATION_STEP: 0.08
  DECELERATION_STEP: 0.1
  //speed set

PID:
  P_GAIN: 0.38
  I_GAIN: 0.000001
  D_GAIN: 0.8

MOVING_AVERAGE_FILTER:
  SAMPLE_SIZE: 30

TOPIC:
  PUB_NAME: /xycar_motor
  SUB_NAME: /usb_cam/image_raw/
  QUEUE_SIZE: 1

CANNY:
  LOW_THRESHOLD: 200
  HIGH_THRESHOLD: 300

HOUGH:
  ABS_SLOPE_RANGE: 3.0
  //10 -> 3
  THRESHOLD: 20
  MIN_LINE_LENGTH: 25
  MAX_LINE_GAP: 10
  // 3 -> 2

DEBUG: true